src/resources.js

```diff
@@ -30,16 +30,17 @@ class Resources {
 	 * Process resources
 	 * @param {Manifest} manifest
 	 */
-	process(manifest){
+	process(manifest) {
 		this.manifest = manifest;
 		this.resources = Object.keys(manifest).
-			map(function (key){
+			map(function (key) {
 				return manifest[key];
 			});
 
 		this.replacementUrls = [];
 
 		this.html = [];
+		//KEM: assets and urls are where the urls in items from the manifest are replaced by "blobs"
 		this.assets = [];
```

Comment:
There's no changes to resources.js other than beautification, but because the audio files are erroring I think the fix will be in how it replaces the URLs in the replacements() function.